### PR TITLE
Feature/428229 LAP CAP retrieve data backend api get

### PR DESCRIPTION
**Pull request by rekhadhankhar2018**

### CODE REVIEW

**Review by udaydefra — COMMENTED** [review]
https://vss-sonarqube-upgrade.azure.defra.cloud/project/issues?pullRequest=36&issueStatuses=OPEN%2CCONFIRMED&id=epr-calculator-api
Only 1 new issue

File: src/EPR.Calculator.API/Mappers/LAPCAPParameterSettingMapper.cs
Comment (udaydefra) [inline src/EPR.Calculator.API/Mappers/LAPCAPParameterSettingMapper.cs:1]: Can we have a unit test for this mapper?

Comment (rekhadhankhar2018) [reply src/EPR.Calculator.API/Mappers/LAPCAPParameterSettingMapper.cs:1]: all done Uday

**Review by johnbenjamin-defra — APPROVED** [review]

**Comment by rekhadhankhar2018** [comment]
another new branch is created for this task this branch is aborted now.

**Pull request closed without merging**